src/components/GPUProviders.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { InventoryItemLoading } from '@openshift-console/dynamic-plugin-sdk';
+import { useTranslation } from 'react-i18next';
+
+import { useAllGpuProviders } from '../resources/nodes';
+import { Dash } from './Dash';
+
+const GPUProviders: React.FC = () => {
```

Comment:
You should be using https://github.com/openshift/console/blob/master/frontend/packages/console-shared/src/components/dashboard/details-card/DetailItem.tsx to make sure we have the same loading/error states.We will need to expose that component (and probably do some slight adjustments)

Reply:
That `DetalItem` is actually used in the `console` when consuming the extension ( https://github.com/openshift/console/pull/11272/files#diff-274cabdee8747fdc07f257b0fb4ea0dc31cd975ac984a3297e48a2870249b5dcR220 )

So do you prefer to expose whole DetailItem instead of it's bits (title + value component)?
With the recent approach, it is up to the consuming code to decide about structure.

Reply:
Yes, I actually prefer exposing `DetailItem` . It gives us more flexibility & we will be able to use loading & error states